README-TESTING.md

```diff
@@ -0,0 +1,32 @@
+# Testing the libraries
+
+The code in these libraries is tested both with deductive verification, using dafny,
+and with traditional runtime testing of compilations by dafny. 
+The latter is important for those cases (e.g. FileIO) where the implementations
+are custom external code for each target programming language.
+Runtime tests are also a check that dafny compiled the (verified) dafny code correctly.
```

Comment:
(minor) I'd be slightly happier reversing the order of these two motivations, just because all Dafny code needs tests because of the second reason, whereas the first is just emphasizing the code most likely to be among the riskiest inside the trusted computing base.

Reply:
There's also the reason of making sure we watch out for any code that isn't currently supported for all compilation targets (but we can leave that for now in this PR, since we're not yet claiming anything about this)

Reply:
Switched the order.